presidio-api/cmd/presidio-api/docs/recognyzer_swagger.go

```diff
@@ -0,0 +1,68 @@
+package docs
```

Comment:
Is this for the recognizer store?
there's a typo in the file name (recognyzer instead of recognizer). If this is for the store, it's worth calling it recognizer_store_swagger

Reply:
Fixed the file name.
This refers to the CRUD operations of recognizers. Do you think it should be renamed to recognizer_store_swagger?